src/NuGet.Services.BasicSearch/Startup.cs

```diff
@@ -246,7 +246,7 @@ private bool InitializeSearcherManager(IndexingConfiguration config, Directory d
                     shouldRetry: e =>
                     {
                         // Retry on any exception (but log it)
-                        _logger.LogError("Startup: An error occurred initializing searcher manager. Going to retry...",
+                        _logger.LogError("Startup: An error occurred initializing searcher manager. Going to retry... Exception: {Exception}",
                             e);
                         _searchTelemetryClient.TrackMetric(SearchTelemetryClient.MetricName.SearchIndexReopenFailed, 1);
```

Comment:
This goes to AI's trace table due to the configuration set up by ServerCommon on this [line](https://github.com/NuGet/NuGet.Services.Metadata/blob/0fc73fd1f138abce259e6e40ef98def52cd1a992/src/NuGet.Services.BasicSearch/Startup.cs#L86). From local testing, it does not go to the `exceptions` table though.